Source/rclUE/Public/rclcUtilities.h

```diff
@@ -139,3 +147,113 @@ static const TMap<TEnumAsByte<UROS2QoS>, rmw_qos_profile_t> QoS_LUT = {
     {UROS2QoS::ParameterEvents, rmw_qos_profile_parameter_events},
     {UROS2QoS::System, rmw_qos_profile_system_default},
     {UROS2QoS::UnknownQoS, rmw_qos_profile_unknown}};
+
+UCLASS()
+class UROS2Utils : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+public:
+    static FString StringROSToUE(const rosidl_runtime_c__String& InStr)
+    {
+        FString outStr;
+        outStr.AppendChars(InStr.data, InStr.size);
+        return outStr;
+    }
+    static TArray<FString> StringSequenceROSToUE(const rosidl_runtime_c__String__Sequence& InStrSequence)
+    {
+        TArray<FString> outStrArray;
+        for (size_t i = 0; i < InStrSequence.size; ++i)
+        {
+            FString outStr;
+            outStr.AppendChars(InStrSequence.data[i].data, InStrSequence.data[i].size);
+            outStrArray.Emplace(MoveTemp(outStr));
+        }
+        return outStrArray;
+    }
+
+    static int32 GetStringRequiredCapacity(const FString& InStr)
+    {
+        FTCHARToUTF8 strUtf8(*InStr);
+        return (strUtf8.Length() + 1) + sizeof(size_t) + sizeof(size_t);
+    }
+
+    static int32 GetStringArrayRequiredCapacity(const TArray<FString>& InStrList)
+    {
+        int32 totalCapacity = 0;
+        for (const auto& str : InStrList)
+        {
+            totalCapacity += GetStringRequiredCapacity(str);
+        }
+        return totalCapacity;
+    }
+
+    static void SetStringProp(rosidl_runtime_c__String& OutStr, const FString& InStr)
```

Comment:
'Prop' is for Properties?
Putting the 'In' variable before the 'Out' one in the parameter list seems more logical to me. Or is this a common way to do in ros or ue code?